buildlib/pr/codestyle.yml

```diff
@@ -70,3 +70,22 @@ jobs:
               azure_complete_with_issues "Code is not formatted according to the code style, see $url for more info."
           fi
         condition: eq(variables['Build.Reason'], 'PullRequest')
+
+  - job: codespell
+    displayName: codespell check
+    pool:
+      name: MLNX
+      demands:
+      - ucx_docker -equals yes
+    container: fedora
+    steps:
+      - checkout: self
+        clean: true
+        fetchDepth: 100
+        retryCountOnTaskFailure: 5
+
+      - bash: |
+          sudo pip3 install codespell
```

Comment:
maybe instead add those commands as a shell script so that we can run it on the development machine too?

Reply:
Now these commands take 4 lines, and it can be reduced to two lines (pip install + codespell run). Do you think that running a script would significantly simplify running on development machine against running two commands?

Reply:
up to you, but it cannot hurt, specially since scripts start small and grow with the time.

Reply:
can we make it part of the docker image?
or use VENV like here: https://github.com/openucx/ucx/blob/d346ea2c58b1aa40a882e0b94a46ace43aa0d647/buildlib/tools/perf-common.yml#L31-L35
To avoid changing the root file system

Reply:
Yes, we can, but what is the problem with installing package inside container?

Reply:
IMO it may be error prone to be run accidentally without container

Reply:
Done